pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix

```diff
@@ -151,6 +151,7 @@
   QuartzCore                       = { inherit ApplicationServices CoreImage CoreVideo Metal OpenCL libobjc; };
   QuickLook                        = { inherit ApplicationServices; };
   QuickLookThumbnailing            = {};
+  QuickTime                        = {}; # it doesn't exist
```

Comment:
Slightly hard to convey in text what I had in mind. Here's the patch I had in mind.

```diff
diff --git a/pkgs/os-specific/darwin/apple-sdk-11.0/apple_sdk.nix b/pkgs/os-specific/darwin/apple-sdk-11.0/apple_sdk.nix
index 3a20846276f..638ab86fb52 100644
--- a/pkgs/os-specific/darwin/apple-sdk-11.0/apple_sdk.nix
+++ b/pkgs/os-specific/darwin/apple-sdk-11.0/apple_sdk.nix
@@ -147,6 +147,10 @@ in rec {
     CoreFoundation = lib.overrideDerivation super.CoreFoundation (drv: {
       setupHook = ./cf-setup-hook.sh;
     });
+
+    # This framework doesn't exist in newer SDKs (somewhere around 10.13), but
+    # there are references to it in nixpkgs.
+    QuickTime = throw "QuickTime framework not available";
   };

   bareFrameworks = (
diff --git a/pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix b/pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix
index 5b41d571de2..c8f8ccc4a08 100644
--- a/pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix
+++ b/pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix
@@ -151,7 +151,6 @@
   QuartzCore                       = { inherit ApplicationServices CoreImage CoreVideo Metal OpenCL libobjc; };
   QuickLook                        = { inherit ApplicationServices; };
   QuickLookThumbnailing            = {};
-  QuickTime                        = {}; # it doesn't exist
   RealityKit                       = {};
   ReplayKit                        = {};
   Ruby                             = {};```